releasenotes/v0.10.0.md

```diff
@@ -0,0 +1,115 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v0.9.0
+
+## :warning: Breaking Changes
+
+- Bump Kubernetes api, controller-runtime and decouple webhooks from api (#2390)
+- Only accept HTTPHeadersRef in same namespace [GHSA-c98h-7hp9-v9hq](https://github.com/metal3-io/baremetal-operator/security/advisories/GHSA-c98h-7hp9-v9hq)
+- Remove keepalived from BMO repository (#2228)
+
+## :sparkles: New Features
+
+- Use standard GetVirtualMedia API for checking dataImage status (#2336)
+- Add support for Disablepoweroff (#2229)
+- add support for locally built ironic in e2e test (#2167)
+
+## :bug: Bug Fixes
+
+- Fix few errors when making beta release notes (#2381)
+- Handle dataImage before reboot annotation to prevent annotation removal before all actions are complete (#2333)
+- fix broken release process (#2325)
+- Define some IPA (downloader) related variables (#2264)
+- bump x/net, x/crypto and x/oauth2 (#2316)
+- update rescue iso used for e2e tests (#2315)
+- Add BMH OwnerReference for DataImage right after its creation (#2249)
+- bump golang to 1.23.6 (#2243)
+- Set ownerReference for dataImage during BMH Reconcile (#2238)
+- Do not create HFC for non-redfish BMH (#2227)
+- bump golang to 1.23.5 (#2224)
+- Set e2e timeout to 1.5h (#2191)
```

Comment:
This doesnt look like a bug fix either

Reply:
Good point, I missed this one. It should be other but was marked incorrectly.

Reply:
removed